components/compounds/DrawerMenu.tsx

```diff
@@ -14,7 +14,7 @@ export const DrawerMenu = ({ children, className }: Props) => {
 
   return (
     <div className={ className }>
-      <IconButton onClick={ toggleDrawer }>
+      <IconButton onClick={ toggleDrawer } aria-label='Toggle Menu'>
```

Comment:
Is it ok for the aria-labels to only be in English? (just curious)

Reply:
This should def be in JP as well. I didn't add the translations, but I should consider adding an accessability translations option as well

Reply:
Ok